### ISSUE TITLE
Reverse numbering

### ISSUE THREAD

**Issue by jbezos** [jbezos/enumitem#4]
From

https://tex.stackexchange.com/questions/153380/enumitem-newlist-with-etaremune

I cannot use etaremune with enumitem. Does anyone know any other way to do it? Basically a reverse numbered list with flexibility of defining the way it looks (like square brackets, indentations, separations etc.

**Comment by MLopez-Ibanez** [comment]
An implementation: https://tex.stackexchange.com/a/627598

Would you accept a pull request?

**Comment by jbezos** [comment]
I’m closing this issue because it’s an enhancement request. This doesn’t mean it’s rejected, just it’s not listed has an issue.

**Comment by jgmbenoit** [comment]
Actually the example given in the manual does not work on my side (Debian 12).

**Comment by jbezos** [comment]
Which `enumitem`version?

**Comment by jgmbenoit** [comment]
2019/06/20 v3.9

**Comment by jbezos** [comment]
Please, upgrade and see the new example. Current version is 3.11 2025-02-06.

**Comment by jgmbenoit** [comment]
The issue persists.  I attach the sample I used along with the pdf output and the log file.

[enumitem-sample-reversecounting-00-tex.txt](https://github.com/user-attachments/files/21033576/enumitem-sample-reversecounting-00-tex.txt)
 [enumitem-sample-reversecounting-00.log](https://github.com/user-attachments/files/21033528/enumitem-sample-reversecounting-00.log)
[enumitem-sample-reversecounting-00.pdf](https://github.com/user-attachments/files/21033529/enumitem-sample-reversecounting-00.pdf)

**Comment by jbezos** [comment]
@jgmbenoit Definitely, the MWE doesn’t work, indeed. Sadly, I don't have time for the maintenance of `enumitem`. Please, ask on a forum like https://tex.stackexchange.com/. Maybe you can get some help.

**Comment by jgmbenoit** [comment]
Thanks for your feed back. [MLopez-Ibanez](https://github.com/MLopez-Ibanez) gave above a good starting point.
